### DISCUSSION TITLE
gd and ctrl-o will make cusor at center like zz, how to disable this so cursor position stays this same when I go back?

### DISCUSSION THREAD

**Discussion by xzbdmw** [LazyVim/LazyVim#2451] [Q&A]
https://github.com/LazyVim/LazyVim/assets/97848247/b6e769ac-64bf-4fb3-ab0c-f79fd2104740

**Comment by wulymammoth** [comment]
In fact switching to and from another buffer or resizing the terminal will center the line of the active buffer to the window. I don't believe this to be default Neovim or Vim behavior. This bugs me a lot because I have a shell open in another tmux pane and zooming in and out results in this exact behavior. 

@folke thank you so much for your amazing work. If not busy, do you know how this may be disabled and/or configured?

**Reply by xzbdmw** [reply]
I think this is controlled by winresizd autocmd in lazyvim

**Comment by xzbdmw** [comment]
can be turned off by vim.o.jumpoptions=‘view’, I believe this should be default in lazyvim

**Reply by folke** [reply]
For sure! Will add and also unset the default `unload`. I never knew this was configurable.
